strapi/strapi
# [TS] Rename invalid template file with correct extension

### What does it do?

Rename the middleware template file from `middleware.js`.hbs to `middleware.ts.hbs` for TypeScript projects 

### Why is it needed?

When running `yarn strapi generate middleware` on the **v4.3.0-beta.2** release, it fails because it cannot find the file with the correct (ts) extension.

### How to test it?

- Create a TS project
- Run `yarn strapi generate middleware`
- It should successfully generate the middleware

### Related issue(s)/PR(s)

[Issue shared on Strapi's Community Discord](https://discord.com/channels/811989166782021633/953765477308567552/994915015377297448)
